files/image_config/ntp/ntp-systemd-wrapper

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
```

Comment:
Is it related to buster? If not, separate it into another PR?

Reply:
yes, it is. it is due to buster image change from init script to systemd wrapper.